tools/packaging/static-build/agent/Dockerfile

```diff
@@ -19,3 +19,5 @@ RUN apk --no-cache add \
         openssl-libs-static \
         protoc && \
     curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain ${RUST_TOOLCHAIN}
+
+RUN ln -s /usr/bin/aarch64-alpine-linux-musl-gcc "/usr/bin/$(uname -m)-linux-musl-gcc"
```

Comment:
What happens when run this on non-aarch64? Should this be "RUN ln -s /usr/bin/$(uname -m)-alpine-linux-musl-gcc "/usr/bin/$(uname -m)-linux-musl-gcc"?

Reply:
Ha good catch you're right, updating.

Reply:
Should we test this on s390x and powerpc?  As they may not use musl or has different name here.

Reply:
What about only doing this on aarch64 and ignore other archs this way only needed symlinks are created?

Reply:
Great!